application/package-lock.json

```diff
@@ -24,6 +24,13 @@
         "webpack-merge": "^4.1.3"
       }
     },
+    "node_modules/@types/json-schema": {
```

Comment:
Is it necessary to add changes of `package-lock.json` file?

Reply:
Yes , npm run watch didn't work as all dependency were outdated. I had to run npm update -f to update all packages with newer versions compatible with the upgraded NPM packages

Reply:
The problem is simple to reproduce, just download the develop branch, build it and then run : npm run watch
You will get this kind of exceptions : 
`Error: No version of chokidar is available. Tried chokidar@2 and chokidar@3.
You could try to manually install any chokidar version.
chokidar@3: Error: Cannot find module 'chokidar'`

Reply:
Thanks for the explanation 😊